portal-api/src/constants/regex.js

```diff
@@ -0,0 +1,5 @@
+exports.UK_POSTCODE = /^[A-Za-z]{1,2}[0-9Rr][0-9A-Za-z]?[0-9][ABD-HJLNP-UW-Zabd-hjlnp-uw-z]{2}$/;
+
+exports.INDUSTRY_SECTOR_ID = /^[0-9]{1,8}$/;
+
+exports.CODE = /^[a-zA-Z]{2,4}$/;
```

Comment:
can this be COUNTRY_CODE please

Reply:
FWIW https://www.npmjs.com/package/validator seems to have some 2 and 3 digit country code validators, `isISO31661Alpha2` and `isISO31661Alpha3`, not sure if they would work here and what we are using

Reply:
Do we need to be this strict? TBH the API could handle that for us and we just check its not a SSRF attempt